### DISCUSSION TITLE
How to style Code component like chakra-ui website

### DISCUSSION THREAD

**Discussion by bradici** [chakra-ui/chakra-ui#3531] [Q&A] [answered]
Hello everyone!

I want to style the code of my blog posts using the element "Code" .
Im looking to have the syntax highlighted depending on the programming language just as you will do with Prismjs.
Im not sure if Chakra-UI supports that, in the [docs](https://chakra-ui.com/docs/data-display/code#colors) I only found that you can change colorScheme but is not what im looking for. 

I attach an example of how code is display in the chakra-ui website which is what I am aiming for. 

![image](https://user-images.githubusercontent.com/68830100/110526608-c1feaa00-80f4-11eb-9a53-76b10b9693fa.png)

**Comment by chr-ge** [comment, marked as answer]
They are using [prism-react-renderer](https://github.com/FormidableLabs/prism-react-renderer) with the "prism-react-renderer/themes/nightOwl" theme. Is that what you are asking for?

**Reply by bradici** [reply]
Yes, I was wondering if Chakra had a component like the one called "Code" but for block code instead of in-line. 
But if they are using that, I asume there isn't one yet. Thank you for your reply! Using prism-react-renderer will do for what I need to do.
